biggerquery/cli.py

```diff
@@ -229,9 +230,51 @@ def _parse_args(project_name: Optional[str], args) -> Namespace:
     _create_deploy_image_parser(subparsers)
     _create_deploy_parser(subparsers)
 
+    _create_build_dags_parser(subparsers)
+    _create_build_image_parser(subparsers)
+    _create_build_package_parser(subparsers)
+    _create_build_parser(subparsers)
+
     return parser.parse_args(args)
 
 
+def _create_build_parser(subparsers):
+    parser = subparsers.add_parser('build', description='Builds docker image, DAG files and .whl package from local files.')
+    _add_build_dags_parser_arguments(parser)
+    _add_build_image_parser_arguments(parser)
+
+
+def _create_build_package_parser(subparsers):
+    subparsers.add_parser('build-package', description='Builds .whl package from local files.')
+
+
+def _add_build_dags_parser_arguments(parser):
+    parser.add_argument('-w', '--workflow',
+                        type=str,
+                        help='The id of the workflow to start.')
+    parser.add_argument('-t', '--start-time',
+                        help='The date and time for which workflow should work.')
```

Comment:
'Point of time from which a workflow should start work.'
'For workflows triggered hourly -- datetime in format: ..., for example ...
'For workflows triggered daily -- date in format: ..., for example ...

Reply:
I think for daily workflows we can pass datetime too

Reply:
We need datetime examples here

Reply:
hourly workflow 2020-07-01 10:00:00
daily workflow 2020-07-01

Reply:
Done